raiden/api/v1/resources.py

```diff
@@ -95,13 +79,45 @@ def get(self, **kwargs):
         return self.rest_api.get_partners_by_token(**kwargs)
 
 
-class EventsResoure(BaseResource):
-    """no args, since we are filtering in the client for now"""
+class NetworkEventsResoure(BaseResource):
 
-    _route = '/api/events'
+    get_schema = EventRequestSchema()
 
     def __init__(self, **kwargs):
-        super(EventsResoure, self).__init__(**kwargs)
+        super(NetworkEventsResoure, self).__init__(**kwargs)
 
-    def get(self):
-        return self.rest_api.get_new_events()
+    @use_kwargs(get_schema, locations=('query',))
+    def get(self, **kwargs):
+        return self.rest_api.get_network_events(kwargs['from_block'], kwargs['to_block'])
+
+
+class TokenEventsResource(BaseResource):
+
+    get_schema = EventRequestSchema()
+
+    def __init__(self, **kwargs):
+        super(TokenEventsResource, self).__init__(**kwargs)
+
+    @use_kwargs(get_schema, locations=('query',))
+    def get(self, *args, **kwargs):
```

Comment:
Where does the `token_address` come from? Can it be overwritten by a query string parameter? I believe this is a bug and that we should use the positional value to get the `token_address` defined in the URL `/events/token/<hexaddress:token_address>`.

Could we have a unit test to make sure that a query string does *not* overwrite the address defined in the URL path?